requirements.txt

```diff
@@ -13,7 +13,7 @@ Markdown
 msgpack>=0.5.0
 psycopg2
 python_dateutil
-PyYAML
+PyYAML>=5.1
```

Comment:
this forces an upgrade of PyYAML in the docker image (which should use Debian packages from Debian buster). Also `full_load` is only available in that version. Please don't do this version bump, and replace `load` with `safe_load` instead of `full_load` in the code. This way you both make the PyYAML usage safer in a way that works with both PyYAML 3 and PyYAML 5, and will also fix this warning.

Reply:
I replaced most calls for `safe_load`. The only ones left out were ProjectStatus `fixes` and `regressions` because they use python deserializer (due to OrderedDict), which behaves like a threat for `safe_load`, breaking it. Instead of changing `fixes` and `regressions`, I chose to keep their old `load` function, just passing the default `Loader` to it, thus getting rid of the warning. Also I've changed tests a bit to make sure they're loading correctly.